### PR TITLE
Engagement Banner test for ROW

### PR DESCRIPTION
**Pull request by svillafe**
## What does this change?
Adds engagement banner test for ROW.

### Variant
picture 664

### Control
picture 665

## Tested in CODE?
Not yet

### CODE REVIEW

**Review by Mullefa — APPROVED** [review]

**Comment by svillafe** [comment]
Tested in CODE and is working ✅

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-engagement-banner-row-support.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const INTsupportURL = 'https://support.theguardian.com/int';
+
+export const AcquisitionsEngagementBannerRowSupport: AcquisitionsABTest = {
+    id: abTestName,
+    campaignId: abTestName,
+    start: '2018-03-01',
+    expiry: '2018-05-24',
+    author: 'Santiago Villa Fernandez',
+    description:
+        'Points the "support the guardian" link in the engagement banner to the aud version of the support site',
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-engagement-banner-row-support.js]: copy/paste hangover: int version, not aud version

Comment (svillafe) [reply static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-engagement-banner-row-support.js]: 😅

**Review by joelochlann — APPROVED** [review]
🌎 🌍 🌏

**Pull request merged**